src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -109,12 +112,19 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
     bbox: BBox,
     fromTime: Date,
     toTime: Date,
-    maxCount: number = 50,
-    offset: number = 0,
+    maxCount: number | null = null,
+    offset: number | null = null,
+    reqConfig?: RequestConfiguration,
   ): Promise<PaginatedTiles> {
     if (!this.dataset.searchIndexUrl) {
       throw new Error('This dataset does not support searching for tiles');
     }
+    if (maxCount === null) {
+      maxCount = DEFAULT_FIND_TILES_MAX_COUNT_PARAMETER;
+    }
+    if (offset === null) {
+      offset = DEFAULT_FIND_TILES_OFFSET_PARAMETER;
```

Comment:
Minor nitpick: `DEFAULT_FIND_TILES_OFFSET_PARAMETER` is not needed, it is clearer if it is set to `0` directly (because any other value of the constant doesn't make sense). Fftr.